src/query/sensor/models/offset-body.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsInt, IsNotEmpty } from 'class-validator';
+
+export class OffsetBody {
```

Comment:
Deze is toch ook gelijk aan de class in de owner package?

Reply:
Yes, maar ik weet niet of het mooier is om een extra module te maken voor files die 2x gebruikt worden (in sensor en owner). Nu is het wel netjes opgedeeld in modules. Heb jij daar ervaring mee? Hoe zou jij dat aanpakken?

Reply:
Ik zou een common package maken waarin gedeelde sources zitten. Copy-paste is tegen het DRY principe: Don't Repeat Yourself. Da's meestal niet zo goed idee ... totdat je de verkeerde of 'belastende' koppeling gaat maken tussen delen die totaal geen relatie hebben. Dat is in dit geval niet zo. Sterker nog: een `DomainException`  zou ik verwacht dat _juist_ gedeeld wordt binnen deze component. Kijk, hij hoeft niet ook nog gedeeld te worden met de sync of frontend bijvoorbeeld. Maar in deze packages zou ik dat wel doen.